.gitignore

```diff
@@ -8,6 +8,9 @@
 .buildlog/
 .history
 .svn/
+*.env
+*.env.development
+*.env.production
```

Comment:
Let's remove these from the ignore for now. In future, if we have something to hide in them we'll have them removed.